snuba/api.py

```diff
@@ -262,17 +263,22 @@ def json_default(obj):
 
 @split_query
 def parse_and_run_query(dataset, request: Request, timer):
-    from_date, to_date = get_time_limit(request.extensions['timeseries'])
+    from_date, to_date = TimeSeriesExtensionProcessor.get_time_limit(request.extensions['timeseries'])
```

Comment:
Doesn't _need_ to happen here, but we'll want to remove this in the near future. If you want to remove this (and the stat that uses it, as well as making `get_time_limit` private), fine by me.

Reply:
Manu is already revisiting stats in his work on the performance extension. I think I will let him go ahead.

Reply:
:+1:

I'll probably do it after the performance extension stuff (and after I reach a stopping point for outcomes cause I want to get that out first), but yea, I'm going to start thinking about how we can refactor stats.